Libraries/LibWeb/HTML/PopoverInvokerElement.idl

```diff
@@ -0,0 +1,11 @@
+[MissingValueDefault=toggle, InvalidValueDefault=toggle]
+enum PopoverTargetActionAttribute {
+    "toggle",
+    "show",
+    "hide"
+};
+
+interface mixin PopoverInvokerElement {
```

Comment:
Could we add a spec link here: https://html.spec.whatwg.org/multipage/popover.html#popoverinvokerelement